src/com/magento/idea/magento2plugin/actions/context/xml/NewWebapiXmlAction.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+package com.magento.idea.magento2plugin.actions.context.xml;
+
+import com.intellij.ide.fileTemplates.actions.AttributesDefaults;
+import com.intellij.psi.PsiDirectory;
+import com.intellij.psi.PsiFile;
+import com.magento.idea.magento2plugin.actions.context.AbstractContextAction;
+import com.magento.idea.magento2plugin.magento.files.ModuleWebApiXmlFile;
+import com.magento.idea.magento2plugin.magento.packages.ComponentType;
+import com.magento.idea.magento2plugin.magento.packages.Package;
+import com.magento.idea.magento2plugin.util.magento.GetMagentoModuleUtil;
+import org.jetbrains.annotations.NotNull;
+
+public class NewWebapiXmlAction extends AbstractContextAction {
+
+    public static final String ACTION_NAME = "Magento 2 WEBAPI File";
+    public static final String ACTION_DESCRIPTION = "Create a new Magento 2 webapi.xml file";
+
+    /**
+     * New routes.xml file generation action constructor.
```

Comment:
Please, change the annotation to the appropriate one.

Reply:
Hi @bohdan-harniuk .
Thanks for your code review. I fixed. Please check.